src/components/manage/Controlpanels/Controlpanel.jsx

```diff
@@ -141,6 +141,35 @@ class Controlpanel extends Component {
   }
   form = React.createRef();
 
+  // Goal would be to replace the Form schema prop in line 182, with this function
+  filterSchema = (controlpanel) => {
+    const panelType = controlpanel['@id'].replace('/@controlpanels/', '');
```

Comment:
@danalvrz The `@id` may be a complete URL, not just a path. I checked and on the demo site it is e.g. `https://6.demo.plone.org/@controlpanels/date-and-time`. So I would try to get the text after the last slash, rather than replacing `/@controlpanels/`